SonarScanner.Shim/Resources.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Ce code a été généré par un outil.
```

Comment:
Nope. Are you using French localized version of VS, or French Windows language pack? Perhaps changing the Windows display language could help, but I never tried that...